pkg/loop/config.go

```diff
@@ -51,6 +54,8 @@ type EnvConfig struct {
 	TelemetryTraceSampleRatio   float64
 	TelemetryAuthHeaders        map[string]string
 	TelemetryAuthPubKeyHex      string
+	EmitterBatchProcessor       bool
+	EmitterExportTimeout        time.Duration
```

Comment:
should these fields be consumed by the beholder client?

Reply:
yes